src/quo2/components/shell/scan.cljs

```diff
@@ -0,0 +1,106 @@
+(ns quo2.components.shell.scan
+  (:require [quo2.foundations.colors :as colors]
+            [quo.react :as react]
+            [reagent.core :as reagent]
+            [quo.react-native :as rn]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.reanimated :as reanimated]
+            [status-im.utils.dimensions :as dimensions]))
+
+(defn corner [outside-border]
+  (let [{:keys [width height]} (dimensions/window)]
+    [rn/view {:style (merge {:border-color     colors/white
+                             :border-width     2
+                             :width            (* 0.20 width)
+                             :height           (* 0.12 height)
+                             :position         :absolute
+                             :background-color :transparent}
+                            (case outside-border
+                              :top-left {:top                    -2
+                                         :left                   -2
+                                         :border-top-left-radius 16
+                                         :border-right-width 0
+                                         :border-bottom-width 0}
+                              :top-right {:top                     -2
+                                          :right                   -1
+                                          :border-top-right-radius 16
+                                          :border-left-width 0
+                                          :border-bottom-width 0}
+                              :bottom-left {:bottom                    -1
+                                            :left                      -2
+                                            :border-bottom-left-radius 16
+                                            :border-right-width 0
+                                            :border-top-width 0}
+                              :bottom-right {:bottom                     -1
+                                             :right                      -1
+                                             :border-bottom-right-radius 16
+                                             :border-left-width 0
+                                             :border-top-width 0}))}]))
+
+(def viewfinder-port
+  {:position        :absolute
+   :align-items     :center
+   :justify-content :center
+   :flex            1})
+
+(defn- viewfinder [size border-radius flashlight-on?]
+  [:f>
+   (fn []
+     [rn/view {:style viewfinder-port}
+      [rn/view {:style {:border-width 700
+                        :border-color "rgba(0,0,0,0.7)"}}
+       [rn/view {:style {:flex-direction :row :justify-content :space-between}}
+        [corner :top-left]
+        [corner :top-right]]
+       [reanimated/view {:style (reanimated/apply-animations-to-style
```

Comment:
hi, why we are using reanimated here? I don't think we are animating scanner anymore.

Reply:
Alisher said we are